openedx/stanford/lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -0,0 +1,66 @@
+from ddt import data
+from ddt import ddt
+from mock import patch
+
+from lms.djangoapps.instructor.tests.test_api import reverse
+from lms.djangoapps.instructor.tests.test_api import TestInstructorAPILevelsDataDump
+from lms.djangoapps.instructor_task.api_helper import AlreadyRunningError
+
+_TASK_DATA = [
```

Comment:
I switched this over to use `ddt`.
Previously, cases were copy/pasted between course_forums/student_forums which led to tech debt/drift. Some times, test cases were missing/forgotten entirely.

Reply:
Now we can run all of the report tasks against all of our report types,
simply by extending this list, no more need to copy/paste test cases.